src/api/api_standalone.py

```diff
@@ -0,0 +1,22 @@
+from fastapi import FastAPI
+import uvicorn
+from fastapi import FastAPI
+from src.api.api_routes import depth_api
+
+import gradio as gr
+
+#TODO very primitive
+
+#TODO add CORS
+
+#TODO enable easy SSL. right now completely unsecured.
```

Comment:
Should be an option - self-signed certs might scare some non-tech-savvy users.

Reply:
self-signed cert are such a headache.

It probably make more sense to make a docker image with some nginx proxy/letsencrypt combo for people who actually want ssl/cors.